core/src/main/java/org/lflang/generator/c/CGenerator.java

```diff
@@ -912,6 +912,12 @@ protected void copyTargetFiles() throws IOException {
           // VS Code configurations
           FileUtil.copyFileFromClassPath("/lib/platform/rp2040/launch.json", vscodePath, true);
         }
+        case STM32 -> {
+          // Copy over STM32 library (Currently hard-coded)
+          FileUtil.copyFileFromClassPath(
+              "/lib/platform/stm32/arm-none-eabi-gcc.cmake", fileConfig.getSrcGenPath(), true);
```

Comment:
Will this work when lfc is just distributed as a JAR?

Reply:
Yes, I believe everything on the classpath should be bundled in the JAR